rustfmt-core/rustfmt-lib/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "rustfmt_lib"
+version = "1.0.0"
+authors = ["topecongiro <seuchida@gmail.com>"]
+edition = "2018"
+description = "Rustfmt as a library"
+license = "Apache-2.0/MIT"
+repository = "https://github.com/rust-lang/rustfmt"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Can probably remove the cargo-generated lines like this right?

Reply:
Yeah, you are right. Good catch, thank you!